runtime-testsuite/test/org/antlr/v4/test/runtime/RuntimeTests.java

```diff
@@ -120,10 +118,20 @@ private static void test(RuntimeTestDescriptor descriptor, RuntimeRunner runner)
 
 		FileUtils.mkdir(runner.getTempDirPath());
 
-		String sourceName = "org/antlr/v4/test/runtime/templates/" + targetName + ".test.stg";
-		String template = RuntimeTestUtils.getTextFromResource(sourceName);
-		STGroup targetTemplates = new STGroupString(sourceName, template, '<', '>');
-		targetTemplates.registerRenderer(String.class, rendered);
+		STGroup targetTemplates = cachedTargetTemplates.get(targetName);
+		if (targetTemplates == null) {
```

Comment:
Just marking this as a similar situation... Not a big deal but multiple loads could occur in a multi threaded environment.